config/purposes/cardinal.yml

```diff
@@ -44,19 +44,19 @@ LCA PBMC:
     - name: "Download Cellaca Input QC1"
       id: cellaca_input_file
       params:
-        - page: 0
+        page: 0
```

Comment:
Thought I'd already merged this fix in.

Reply:
It is weird that it's showing up here as the change is already on the branch I'm trying to merge into.  I think it's because I branched off before your commit and GitHub is trying to be clever and show you what changed in my branch since it was created.  I'll merge the destination branch into mine and I expect they'll magically disappear.